### DISCUSSION TITLE
App cant open virtual xposed v11 android

### DISCUSSION THREAD

**Discussion by 69Johans69** [android-hacker/VAExposed#1043] [Q&A]
Dead by daylight cant open app
Android 11

**Comment by ItzCrazhoi** [comment]
Eu também não consigo sou Android 11 também

**Comment by pegromine** [comment]
Android 12 here. I have the same problem.

**Comment by pegromine** [comment]
Does anyone know which module is recommended for VirtualXposed with game guardian? I can't read Chinese unfortunately. Or Japanese...not sure which language it is.
